presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriverUri.java

```diff
@@ -166,4 +176,22 @@ private void initCatalogAndSchema()
             schema = parts.get(1);
         }
     }
+
+    private static Properties getMergedProperties(URI uri, Properties driverProperties)
+            throws SQLException
+    {
+        Properties result = new Properties();
+        result.putAll(ConnectionProperties.getDefaults());
+        result.putAll(driverProperties);
```

Comment:
driverProperties meaning the ones that are passed into PrestoDriver.connect(). If you have a better name, I'm open to it.